src/Parser/RestorationShaman/Modules/Features/AncestralVigor.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import makeWclUrl from 'common/makeWclUrl';
+import SpellIcon from 'common/SpellIcon';
+
+import LazyLoadStatisticBox from 'Main/LazyLoadStatisticBox';
+
+import Module from 'Parser/Core/Module';
+
+const ANCESTRAL_VIGOR_INCREASED_MAX_HEALTH = 0.1;
+const HP_THRESHOLD = 1 - 1/(1 + ANCESTRAL_VIGOR_INCREASED_MAX_HEALTH);
+// this threshold was used to filter the big damages that has more chance to kill a player if there is no ancestral vigor buff.
+const BIG_DAMAGE_THRESHOLD = 0.2;
```

Comment:
Damage regardless of being small or big would kill you if they bring you below `HP_THRESHOLD` when looking at AV vs no AV. This threshold seems to make it differentiate while I believe that would only be misleading as you'd be just as dead in any case. This also goes for all usages of this threshold.